RELEASE_PROCESS.rst

```diff
@@ -7,9 +7,33 @@ Make the clone using:
 
 ::
 
-     git clone --branch master git@github.com:juju/juju-gui.git
-     cd juju-gui
-     git merge origin/develop
+    git clone git@github.com:juju/juju-gui.git gui-release
+    cd juju-gui
+
+Next you'll want to generate the list of changes for CHANGES.yaml.
+
+::
+
+    git log <last release>...HEAD --author 'jujugui' --format='* [%h] %b'
+
```

Comment:
I just discovered this bit of wizardry:

```
git log `git describe --tags --abbrev=0`..HEAD --author 'jujugui' --format='* [%h] %b'
```

It shows everything since the last tag, which would almost always be what we want. Might make a good Makefile addition.